app/src/lang/translations/en-US.yaml

```
@@ -1364,6 +1364,11 @@ fields:
     custom_aspect_ratios: Custom Aspect Ratios
     theme_light_overrides: Light Theme Customization
     theme_dark_overrides: Dark Theme Customization
+    public_registration: Public registration
+    is_public_registration_enabled: Public registration
```

These should probably both be title case 

```suggestion
    public_registration: Public Registration
    is_public_registration_enabled: Public Registration
```